packages/components/collapse/src/CollapsePanel.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div
+    class="ix-collapse-panel"
+    :class="{
+      'ix-collapse-panel-disabled': disabled,
+      'ix-collapse-panel-borderless': borderless,
+      'ix-collapse-panel-active': isActive,
+    }"
+  >
+    <div
+      class="ix-collapse-panel-header"
+      :class="{
+        'ix-collapse-panel-header-borderless': borderless,
+        'ix-collapse-panel-header-inactive': !isActive,
+        'ix-collapse-panel-header-disabled': disabled,
+      }"
+      @click="headerClick"
+    >
+      <slot name="icon">
+        <ix-icon class="ix-collapse-panel-icon" :name="!isActive ? icons[0] : icons[1]"></ix-icon>
+      </slot>
+
+      <slot
+        name="title"
+        class="ix-collapse-panel-header-content"
+        :class="{ 'ix-collapse-panel-header-borderless-content': borderless }"
+      >
+        {{ title }}
+      </slot>
+    </div>
+    <transition name="ix-slide-down">
+      <div v-show="isActive">
+        <div
+          class="ix-collapse-panel-header-content"
+          :class="{ 'ix-collapse-panel-header-content-borderless': borderless }"
+        >
+          <slot></slot>
+        </div>
+      </div>
+    </transition>
+  </div>
+</template>
+<script lang="ts">
+import { computed, defineComponent, inject } from 'vue'
+import { CollapsePanelProps, subjectInjectKey, collapseInjectionKey } from './types'
+import { PropTypes } from '@idux/cdk/utils'
+import { IxIcon } from '@idux/components/icon'
+
+export default defineComponent({
+  name: 'IxCollapsePanel',
+  components: { IxIcon },
+  props: {
+    name: PropTypes.string.isRequired,
+    title: PropTypes.string,
+    disabled: PropTypes.bool.def(false),
+    icon: PropTypes.arrayOf(PropTypes.string),
+  },
+
+  setup(props: CollapsePanelProps) {
+    const collapse = inject(collapseInjectionKey, {})
+    const subject = inject(subjectInjectKey, {})
+    const icons = computed(() => props.icon ?? ['right', 'down'])
+
+    const isActive = computed(() => {
+      const { name } = props
+      return collapse.active.indexOf(name) > -1
+    })
```

Comment:
```suggestion
    const isActive = computed(() => activeNames.value.includes(props.name))
```
对 props 使用解构会失去响应式，activeNames 是父组件注入的。